.circleci/config.yml

```diff
@@ -1990,7 +1990,7 @@ jobs:
       - setup-python-venv:
           install-python: false
           determined: true
-          extras-requires: "tensorflow==2.12.0 torch==1.9.0"
+          extras-requires: "tensorflow==2.12.0 torch==1.11.0"
           extra-requirements-file: "harness/tests/requirements/requirements-harness.txt"
```

Comment:
Any idea why we have some requirements in the circle config and others in a requirements.txt? That seems weird.

Reply:
Oh, wait. Now I remember. The old pip would install out of order, causing occasional failures. Nevermind, though I think that problem was solved with newer pip.

Reply:
Any updates on this?

Reply:
Ok, I tracked it down: https://hpe-aiatscale.slack.com/archives/C6D7MDSQ5/p1676328721315849?thread_ts=1676326807.488339&cid=C6D7MDSQ5

We originally did this because pip at one time didn't install requirements in order.  That behavior was fixed in pip 6.1.0.  Deepspeed also did not originally specify requirements properly, though, so pip couldn't resolve the need to install torch first.  _That_ was fixed in deepspeed 0.5.9.

So as long as we specify pip > 6.1.0 (which is kinda old) and deepspeed >= 0.5.9, we should be able to move torch into requirements.txt where they belong.  I'd assume tensorflow as well.

Reply:
Moved requirements over to requirements.txt files. Additionally, noticed we have different python versions between our `.circleci` venv and our `.github` venv, so I corrected that.